clusters/app.ci/assets/admin_ci-monitoring-view_rbac.yaml

```diff
@@ -1,12 +0,0 @@
-apiVersion: rbac.authorization.k8s.io/v1
-kind: ClusterRoleBinding
-metadata:
-  name: ci-monitoring-view
```

Comment:
After merge:

```console
oc delete ClusterRoleBinding ci-monitoring-view --context app.ci
```

This one is replaced by `clusters/build-clusters/common/monitoring_viewer.yaml`.